requirements_dev.txt

```diff
@@ -23,6 +23,7 @@ plotly
 bokeh
 
 tqdm
+tenacity
```

Comment:
Do you want to add some constraints here?

Reply:
no as the are no constraints in .[importers] in pyproject.toml